### ISSUE TITLE
Zone hysteresis

### ISSUE THREAD

**Issue by sandreialexandru** [pihome-shc/pihome#427]
Zone hysteresis value is not calculated by boiler.php. Having zone and boiler hysteresis is redundant and I suggest deleting column in zone_sensors table and from the GUI.

**Comment by sandreialexandru** [comment]
Same logic applies to Maximum Operation Time

**Comment by pihome-shc** [comment]
same comments as Adding boost #426 re: zone hysteresis and max running time. if we dont see this any useable we can delete them.

**Comment by sandreialexandru** [comment]
Zone hysteresis is redundant with boiler hysteresis. Let's say that a zone runs for 60 minutes, then the boiler stops. Another zone requires heating in the next 3 minutes but boiler won't start untill the hysteresis passes. Zone hysteresis would only be valuable if only one zone is used.

Setting hysteresis for zones doesn't matter as  long as boiler could run continuously for different zones. Hysteresis is a boiler property, by definition.

**Comment by pihome-shc** [comment]
good point, i was thinking in frost protection mode boiler will run indefinitely, may be we need to introduce some time limit on frost protection then, and drop zone hysteresis and zone max operation time. 
what other think?

**Comment by sandreialexandru** [comment]
What if time ends for frost protection? Then no protection would occur. Taking a look at other thermostats I see no time limit for frost protection. The logic is that water will not freeze if 2°-3°C are above the set temperature, not to mention that the water circulating in the system will have 60°C or more.

**Comment by pihome-shc** [comment]
i was thinking to make system more intelligent, i.e if boiler is running for while and there is not temperature change then something else is wrong and shutdown the frost protection and one way could be offer time limit for boiler run for frost protection mode or check if temperature is not changing after 10 15 minuts

**Comment by sandreialexandru** [comment]
Nevertheless zone hysteresis and zone maximum operation time are not needed in my opinion. Those properties are for boiler only.

**Comment by pihome-shc** [comment]
Agree, one second thought @twa127 have you used maximum operation time for non heating zone?

**Comment by twa127** [comment]
Hi,

No I don't think it is used anywhere

**Comment by pihome-shc** [comment]
then its safe to remove as hysteresis and zone maximum operation time are not needed

**Comment by pihome-shc** [comment]
@sandreialexandru any further action required? can we close this issue?
